rnd/autogpt_server/autogpt_server/server/server.py

```diff
@@ -147,19 +152,40 @@ def execution_manager_client(self) -> ExecutionManager:
     def execution_scheduler_client(self) -> ExecutionScheduler:
         return get_service_client(ExecutionScheduler)
 
-    def get_graph_blocks(self) -> list[dict[Any, Any]]:
-        return [v.to_dict() for v in block.get_blocks()]
+    @classmethod
+    def handle_internal_error(cls, request, exc):
+        return JSONResponse(
+            content={
+                "message": f"{request.url.path} call failure",
+                "error": str(exc),
+            },
+            status_code=500,
+        )
+
+    @classmethod
+    def get_graph_blocks(cls) -> list[dict[Any, Any]]:
+        return [v.to_dict() for v in block.get_blocks().values()]
+
+    @classmethod
+    def execute_graph_block(cls, block_id: str, data: dict[str, Any]) -> list:
+        obj = block.get_block(block_id)
+        if not obj:
+            raise HTTPException(status_code=404, detail=f"Block #{block_id} not found.")
+        return [{name: data} for name, data in obj.execute(data)]
 
-    async def get_graphs(self) -> list[str]:
+    @classmethod
+    async def get_graphs(cls) -> list[str]:
```

Comment:
Why are we changing these to class methods?

Reply:
linter keeps asking to take the function out of the class or make it static.